app/components/form/step_navigation_component.html.erb

```diff
@@ -0,0 +1,34 @@
+<div>
+    <%# Back navigation %>
+    <%= button_tag type: :submit,
+        name: :nav_step,
+        value: @nav_back,
+        class: "btn btn-link #{@nav_back.nil? ? 'disabled' : 'enabled'}",
+        title: "Back step",
+        aria: { label: "Back step" },
+        disabled: !@nav_back do %>
+        <% if @nav_back.present? %>
+            <%= link_to @nav_back, title: "Back step", aria: { label: "Back step" } do %>
```

Comment:
Chose to do a link within the button to have access to visible display of route/url on hover